frontend/src/pages/learningCenter/LearningCenterToolbar.tsx

```diff
@@ -131,16 +136,17 @@ const LearningCenterToolbar: React.FC<LearningCenterToolbarProps> = ({
     </SelectOption>
   ));
   const onSortOrderSelect = React.useCallback(
-    (e: React.MouseEvent | React.ChangeEvent) => {
+    (_e: React.MouseEvent<Element, MouseEvent> | undefined) => {
       setIsSortOrderDropdownOpen(false);
-      const selection = e.target instanceof Element ? e.target.getAttribute('data-key') ?? '' : '';
+      const selection =
+        _e?.currentTarget instanceof Element ? _e.currentTarget.getAttribute('id') ?? '' : '';
       setQueryArgument(navigate, DOC_SORT_ORDER_KEY, selection);
     },
     [navigate],
   );
 
   const sortOrderDropdownItems = Object.entries(sortOrders).map(([key, val]) => (
-    <SelectOption key={key} data-key={key} value={val} data-testid={key}>
+    <SelectOption key={key} id={key} data-key={key} value={val} data-testid={key}>
```

Comment:
@christianvogt this is necessary because data-key attribute is not found in target event

Reply:
Use the 2nd param from `onSelect` to get the value:

`(property) SelectProps.onSelect?: ((event?: React.MouseEvent<Element, MouseEvent> | undefined, value?: string | number | undefined) => void) | undefined`

Reply:
@christianvogt I did tried that, It works fine for sortTypeDropdown since the key and value are both same, but sortOrderDropdown has different key and value

Reply:
If we aren't using the value, then set the value to be the key. Or correct the enum values. No point to not use the component as intended with the value.